terraform/staging.tfvars

```diff
@@ -56,16 +56,17 @@ website_google_tag_id = "G-D5DFR7BN0N"
 cluster_container_insights_enabled = true
 
 // API / Backend
-api_enabled                           = true
-api_default_desired_task_count        = 1
-api_minumum_task_count                = 1
-api_maximum_task_count                = 5
-api_enable_grants_scraper             = false
-api_enable_grants_digest              = false
-api_enable_new_team_terminology       = true
-api_enable_my_profile                 = true
-api_enable_saved_search_grants_digest = true
-api_log_retention_in_days             = 14
+api_enabled                            = true
+api_default_desired_task_count         = 1
+api_minumum_task_count                 = 1
+api_maximum_task_count                 = 5
+api_enable_grants_scraper              = false
+api_enable_grants_digest               = false
+api_enable_new_team_terminology        = true
+api_enable_my_profile                  = true
+api_enable_saved_search_grants_digest  = true
+api_enable_grant_digest_scheduled_task = false
```

Comment:
Do we want this to be flipped to true in staging so we can test the functionality as soon as it lands?

Reply:
Yes please!